### PR TITLE
Burningwater202

### PR DESCRIPTION
**Pull request by burningwater202**
I have no idea what I'm doing

### CODE REVIEW

File: src/main/java/com/example/examplemod/ExampleMod.java
Comment (HbmMods) [inline src/main/java/com/example/examplemod/ExampleMod.java:1]: please don't include this

File: src/main/resources/assets/hbm/lang/en_US.lang
Comment (HbmMods) [inline src/main/resources/assets/hbm/lang/en_US.lang:1]: encoding machine broke

**Comment by HbmMods** [comment]
I'm just going to delete the offending files, I hope the delete option works the way I think it does because github's documentation is piss poor and most gitoids aren't exactly helpful either.

**Comment by HbmMods** [comment]
nope it doesn't do that at all

**Comment by burningwater202** [comment]
GitHub my behated

**Comment by HbmMods** [comment]
ow my legs

**Pull request merged**